.gitignore

```diff
@@ -1,2 +1,2 @@
-/node_modules
-/test.md
\ No newline at end of file
+
+test/
```

Comment:
- [ ] According to requirements, you should have 2 files here which are `node_modules` and `test.md`. Kindly check and ensure you have these files here. `.gitignore` is used to hide files that are of no significant importance to visitors or users. This also helps to keep our directory clean. :smile: